File: tp.rb
villes = ["Paris", "New York", "Berlin", "Montreal"]

voyages = [
  { ville: "Paris", duree: 10 },
  { ville: "New York", duree: 5 },
  { ville: "Berlin", duree: 2 },
  { ville: "Montreal", duree: 15 }
]

#######################################################
puts "-----------------------------------------------"
puts "DEFI N°1 - Si j'étais en vacances, j'irais à..." 


villes.each do |ville|
  puts ville
end

#######################################################
puts "-----------------------------------------------"
puts "DEFI N°2 - Détail de mes vacances de rêve" 


voyages.each do |voyage|
  puts "Voyage à #{voyage[:ville]} de #{voyage[:duree]} jours"
end

#######################################################
puts "-----------------------------------------------"
puts "DEFI N°3 - Mes vacances de rêve (enfin presque)" 

voyages.each do |voyage|
  if voyage[:duree] <= 5
    puts "Voyage à #{voyage[:ville]} de #{voyage[:duree]} jours"
  end
end

File: tp.py
villes = ["Bogota", "Lyon", "Cordou", "Berlin", "Londre"]

print("DEFI N°1 - Si j'étais en vacances.. j'irai à : \n")

for l in villes :
	print(l)

voyages = {
	villes[0]:10,
	villes[1]:5,
	villes[2]:7,
	villes[3]:8,
	villes[4]:4,
}

print(voyages)

print("\nDEFI N°2 - Détails de mes vacances : \n")
for l, t in voyages.items():
	print("Voyage à {} pendant {} jours".format(l, t))

print("\nDEFI N°3 - Mes vacances qui ne dure que 5 jours ou moins : \n")
for l, t in voyages.items():
	if t <= 5 :
		print("Voyage à {} pendant {} jours".format(l, t))